index.md

```diff
@@ -54,23 +54,22 @@ bis auf [VSCode](/install/windows_10/#VSCode) (kostenlos, aber proprietär).
   Die Benutzung einer Programmiersprache zur Versuchsauswertung bietet große Vorteile
   gegenüber Alternativen wie Tabellenkalkulationssoftware.
 
+* **Unix/Kommandozeile**: Der Umgang mit der Kommandozeile und mit Unix-artigen
+  Betriebssystemen ist eine wichtige Voraussetzung für wissenschaftliches Arbeiten.
+  Wir bringen euch die Grundlagen bei.
+
 * **Git:** Reproduzierbarkeit und kollaboratives Arbeiten sind zwei Hauptaspekte
   des guten wissenschaftlichen Arbeitens.
   Git ist das Standardwerkzeug für Versionskontrolle,
   welche es ermöglicht jederzeit zu älteren Versionen zurückzukehren,
   Änderungen nachzuvollziehen und gemeinsam an Projekten zu arbeiten.
 
-* **Make**: Das Standardtool zur automatisierten Ausführung von komplexeren Aufgaben.
+  * **Make**: Das Standardtool zur automatisierten Ausführung von komplexeren Aufgaben.
```

Comment:
Maybe rephrase to:
- Eins der standard-Tools zur ...
- Unser favorisiertes standard-tool zur ...